mariadb/mariadb_cursor.c

```diff
@@ -1135,7 +1135,7 @@ MrdbCursor_execute_text(MrdbCursor *self, PyObject *stmt)
         statement = PyUnicode_AsUTF8AndSize(stmt, (Py_ssize_t *)&statement_len);
     } else if (Py_TYPE(stmt) == &PyBytes_Type)
     {
-        PyBytes_AsStringAndSize(stmt, &statement, (Py_ssize_t *)&statement_len);
+        PyBytes_AsStringAndSize(stmt, (char**)statement, (Py_ssize_t *)&statement_len);
```

Comment:
Hello. I remove it to convert directly to `char**`.
I tried changing the statement type from `const char*` to `char*` and it also worked.
However, I don't know what is the best way to do it and whether the latter could cause problems.

Reply:
Instead of casting away the `const` incorrectly, within this function you can instead drop `const` from the declaration of `statement`:
```diff
diff --git a/mariadb/mariadb_cursor.c b/mariadb/mariadb_cursor.c
index 65fab74..191f93a 100644
--- a/mariadb/mariadb_cursor.c
+++ b/mariadb/mariadb_cursor.c
@@ -1125,7 +1125,7 @@ MrdbCursor_execute_text(MrdbCursor *self, PyObject *stmt)
 {
     int rc;
     MYSQL *db;
-    const char *statement;
+    char *statement;
     size_t statement_len= 0;
 
     MARIADB_CHECK_CONNECTION(self->connection, NULL);
```

Reply:
Hey. I made these changes and updated the pr. Thanks :)